daemon/protocol/python/scoot/client_lib.py

```diff
@@ -90,13 +93,53 @@ def start():
     raise ScootException(Exception("Could not find domain socket path."))
   if is_started():
     raise ScootException(Exception("Already started."))
-  channel = grpc.insecure_channel("unix://%s" % _domain_sock)
+  channel = None
+  try:
+    channel = grpc.insecure_channel("unix://%s" % _domain_sock)
+  except Exception as e:
+    raise ScootException("Error getting channel:'unix://{}'".format(_domain_sock))
   #channel.subscribe(lambda x: sys.stdout.write(str(x)+"\n"), try_to_connect=True)
```

Comment:
sorry - this was in reference to line 94 (which is commented out), not the error message